addons/godot_xml/xml.gd

```diff
@@ -194,6 +199,9 @@ static func _attach_node_data(node: XMLNode, parser: XMLParser) -> void:
         # we therefore strip "blankets", resulting in only actual content slipping into .content
         node.content = parser.get_node_data().strip_edges().lstrip(" ").rstrip(" ")
 
+static func _attach_node_name(node: XMLNode, parser: XMLParser) -> void:
+    if node.content.is_empty():
+        node.content = parser.get_node_name().strip_edges().lstrip(" ").rstrip(" ")
```

Comment:
CDATA content should be a new attribute on `XMLNode` (`XMLNode.cdata`, for example), to avoid replacing regular textual content with CDATA nodes. There also may be multiple CDATA nodes, so the new property should be an `Array[String]` instead and on each `NODE_CDATA` received CDATA contents should be appended to that array.

Reply:
I agree. Making these changes right now.